docs/cn/Running-Alluxio-On-Docker.md

```diff
@@ -13,7 +13,7 @@ Alluxio可以运行在一个Docker容器中，本指南介绍如何使用Alluxio
 
 # 基础教程
 
-这篇教程主要介绍了如何在单节点上启动docker上的Alluxio
+这篇教程主要介绍如何在单节点上使用 Docker 运行 Alluxio。
```

Comment:
Can you remove the space before and after "Docker" and "Alluxio"?

Reply:
12 / 5000
I have modified it according to your request. thanks.